pythonforandroid/recipe.py

```diff
@@ -574,27 +574,36 @@ def list_recipes(cls, ctx):
     @classmethod
     def get_recipe(cls, name, ctx):
         '''Returns the Recipe with the given name, if it exists.'''
+        name = name.lower()
         if not hasattr(cls, "recipes"):
             cls.recipes = {}
-        if name in cls.recipes:
+        if name in cls.recipes.keys():
```

Comment:
Minor open question: is there any PEP recommandation that recommends being explicit on that specific case or is it just a generic "explicit over implicit"? :thinking:

Reply:
PEP 234 does. https://www.python.org/dev/peps/pep-0234/#dictionary-iterators

the only time I'd use the explicit `keys()` is if I was looping over the keys and modifying somehting (`for x in list(dictname.keys())`), but in this case, the shorter version is notionally clearer, PEP compliant, and very likely faster.